### DISCUSSION TITLE
how to set different qvalue for different clusters when doing peak call

### DISCUSSION THREAD

**Discussion by yingyuan830** [GreenleafLab/ArchR#1537] [Questions / Documentation] [answered]
when doing peak call, it is sometimes necessary to set different qvalue for different clusters with different cell numbers, how can I set them in the function ? can anyone give hints or other suggestions ? Thanks

**Comment by rcorces** [comment, marked as answer]
It is not possible in ArchR to set individual q value cutoffs per cluster. If you want to do something other than what ArchR offers, you can create your own peakset outside of ArchR and use `addPeakSet()` to import it into your project.

**Reply by yingyuan830** [reply]
ok, thanks
